lib/PuppeteerSharp/Launcher.cs

```diff
@@ -156,6 +156,7 @@ public Launcher()
 
                 if (options.LogProcess)
                 {
+                    _processLoaded = true;
```

Comment:
I think this should not be inside the `if` statement

Reply:
true.

Reply:
Done